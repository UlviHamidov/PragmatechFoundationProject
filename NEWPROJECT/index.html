<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>ANA SƏHİFƏ</title>
    <link rel="stylesheet" href="css/style.css">

    <!-- ICONS -->
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.5.0/font/bootstrap-icons.css">
    
    <!-- FONTS -->
    <link href="https://fonts.googleapis.com/css2?family=Oxygen:wght@300&display=swap" rel="stylesheet">
    <link href="https://fonts.googleapis.com/css2?family=Teko:wght@700&display=swap" rel="stylesheet">
</head>
<body>
    <div id="social">
        
        <font>SİTE LOGO | <span>.:Site Slogan:.</span></font>

        <i class="bi bi-search"></i>
        <i class="bi bi-facebook"></i>
        <i class="bi bi-instagram"></i>
        <i class="bi bi-twitter"></i>
        <i class="bi bi-google"></i>
    </div>
    <div id="container">
        <header>
            <img src="img/header.jpg" id="headerimg" alt="">
            <div id="centered">WEBSITE NAME</div>    
        </header>
        
        <section>
            <nav>
                <ul>
                    <li><a href="index.html"><i class="bi bi-house-fill"></i>ANA SƏHİFƏ</a></li>
                    <li><a href="xidmet.html"><i class="bi bi-bookmark-star-fill"></i>XİDMƏTLƏR</a></li>
                    <li><a href="haqqimizda.html"><i class="bi bi-info-circle-fill"></i>HAQQIMIZDA</a></li>
                    <li><a href="qaleriya.html"><i class="bi bi-images"></i>QALERİYA</a></li>
                    <li><a href="elaqe.html"><i class="bi bi-person-lines-fill"></i>ƏLAQƏ</a></li>
                </ul>
            </nav>
            <main>
                <div class="icerisi">
                    <img src="img/1.jpg" alt="" class="mainimg">

                    <h5>NAME CONTENT</h5>
                    <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Sunt doloribus ullam maxime aliquam eveniet odio placeat quas. Doloremque rem omnis minus nostrum neque, qui tempore saepe accusamus blanditiis veniam similique.</p>
                    <a href="#" class="btnmore">Davam...</a>
                </div>

                <div class="icerisi">
                    <img src="img/2.jpg" alt="" class="mainimg">

                    <h5>NAME CONTENT</h5>
                    <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Sunt doloribus ullam maxime aliquam eveniet odio placeat quas. Doloremque rem omnis minus nostrum neque, qui tempore saepe accusamus blanditiis veniam similique.</p>
                    <a href="#" class="btnmore">Davam...</a>
                </div>

                <div class="icerisi">
                    <img src="img/3.jpg" alt="" class="mainimg">

                    <h5>NAME CONTENT</h5>
                    <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Sunt doloribus ullam maxime aliquam eveniet odio placeat quas. Doloremque rem omnis minus nostrum neque, qui tempore saepe accusamus blanditiis veniam similique.</p>
                    <a href="#" class="btnmore">Davam...</a>
                </div>

                <div class="icerisi">
                    <img src="img/4.jpg" alt="" class="mainimg">

                    <h5>NAME CONTENT</h5>
                    <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Sunt doloribus ullam maxime aliquam eveniet odio placeat quas. Doloremque rem omnis minus nostrum neque, qui tempore saepe accusamus blanditiis veniam similique.</p>
                    <a href="#" class="btnmore">Davam...</a>
                </div>
            </main>
        </section>
<footer>BÜTÜN HAQQLAR QORUNUR. &copy;| 2021</footer>
    </div>
</body>
</html>