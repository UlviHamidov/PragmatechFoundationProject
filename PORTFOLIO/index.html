<!DOCTYPE html>
<html lang="az">

<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>ANA SƏHİFƏ | YeniPrayekt</title>
    <link rel="stylesheet" href="css/style.css">
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.5.0/font/bootstrap-icons.css">
    <link rel="preconnect" href="https://fonts.googleapis.com">
    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
    <link href="https://fonts.googleapis.com/css2?family=Nunito:wght@300&display=swap" rel="stylesheet">
</head>

<body>
    <section id="menu">

        <div id="logo">LOGOM</div>
        <nav>
            <a href=""><i class="bi bi-house"></i>ANA SƏHİFƏ</a>
            <a href=""><i class="bi bi-info-square"></i>HAQQIMDA</a>
            <a href=""><i class="bi bi-book"></i>DƏRSLƏRİM</a>
            <a href=""><i class="bi bi-people"></i>KOMANDAM</a>
            <a href=""><i class="bi bi-pin-map"></i>ƏLAQƏ</a>

        </nav>

    </section>

    <section id="mainbg">
        <div id="black">

        </div>

        <div id="maintxt">
            <h2>SLOGAN</h2>
            <hr width="300" align="left">
            <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Praesentium, voluptatum?</p>

        </div>
    </section>

    <section id="hakkimizda">
        <h5>HAKKIMIZDA</h5>
        <div id="container">

            <div id="sol">
                <p id="psol">Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptas unde necessitatibus quod.
                    Omnis itaque ipsa necessitatibus unde praesentium facere earum expedita modi? Accusantium quam
                    blanditiis, sed aperiam mollitia expedita dolorum.</p>
            </div>

            <div id="sag">
                <p id="psag">Lorem ipsum, dolor sit amet consectetur adipisicing elit. Rerum, totam repudiandae, earum
                    quisquam unde cumque illo aliquam explicabo optio dolorum veritatis voluptates quos commodi laborum
                    aspernatur facere, adipisci eos voluptas!</p>
            </div>
            <img src="img/imgbg.jpg" alt="">
        </div>
    </section>

</body>

</html>