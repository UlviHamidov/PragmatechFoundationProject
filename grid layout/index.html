<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Grid</title>
    <link rel="stylesheet" href="css/style.css">
    <link rel="stylesheet" href="css/grid.css">
    <link rel="stylesheet" href="css/layout.css">
    <link rel="stylesheet" href="css/tableStructure.css">
</head>

<body>
   <div class="container">
       <div class="row">
           <div class="col-12">
               <div class="item5">

               </div>
           </div>
       </div>
       <div class="row">
           <div class="col-6">
               <div class="item"></div>
           </div>
           
           <div class="col-6">
               <div class="item2">
                   <h1>Explore the Piece of History</h1>
                   <p>Five centuries later Lorem Ipsum experienced a surge in popularity with the release of Letraset's dry-transfer sheets. These sheets of lettering could be rubbed on anywhere and were quickly adopted by graphic artists, printers, architects, and advertisers for their professional look and ease of use.</p>
                    <p>Letraset included Lorem Ipsum passages in a panoply of fonts, styles, and sizes, solidifying the Latin-esque phrase's place in the print and graphics industry. Those with an eye for detail will have even caught a tribute to the classic text in an episode of Mad Men.</p>
                    <p>The program came bundled with Lorem Ipsum dummy text to help with laying out page content, and other word processors like Microsoft Word followed suit.</p>
                    
                    <p>The program came bundled with Lorem Ipsum dummy text to help with laying out page content,</p>
               </div>
           </div>
        </div>
        <div class="row">
            <div class="col-4">
                <div class="item3"></div>
                <div class="yazi">
                    <h1>dsfdsgds</h1>
                    <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Dolorum perspiciatis fugit quaerat.</p>
                </div>
            </div>
            <div class="col-4">
                <div class="item3"></div>
                <div class="yazi">
                    <h1>dsfdsgds</h1>
                    <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Dolorum perspiciatis fugit quaerat. </p>
                </div>
            </div>
            <div class="col-4">
                <div class="item3"></div>
                <div class="yazi">
                    <h1>dsfdsgds</h1>
                    <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Dolorum perspiciatis fugit quaerat.</p>
                </div>
            </div>
        </div>
        <div class="row">
            <div class="col-4">
                <div class="item3"></div>
                <div class="yazi">
                    <h1>dsfdsgds</h1>
                    <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Dolorum perspiciatis fugit quaerat.</p>
                </div>
            </div>
            <div class="col-4">
                <div class="item3"></div>
                <div class="yazi">
                    <h1>dsfdsgds</h1>
                    <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Dolorum perspiciatis fugit quaerat. </p>
                </div>
            </div>
            <div class="col-4">
                <div class="item3"></div>
                <div class="yazi">
                    <h1>dsfdsgds</h1>
                    <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Dolorum perspiciatis fugit quaerat.</p>
                </div>
            </div>
        </div>

        <div class="row">
            <div class="col-12">
                <div class="item4"> </div>
            </div>
        </div>
   </div>
</body>

</html>