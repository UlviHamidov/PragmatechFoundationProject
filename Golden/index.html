<!doctype html>
<html lang="en">
<head>
  <!-- Required meta tags -->
  <meta charset="utf-8">
  <title>Golden Website</title>
  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">

  <!-- Bootstrap CSS -->



  <link rel="stylesheet" type="text/css" href="css/fontawesome.min.css">
  <link rel="stylesheet" type="text/css" href="css/fontawesome.css">
  <link rel="stylesheet" type="text/css" href="css/all.css">

  <link rel="stylesheet" type="text/css" href="css/bootstrap.min.css">
  <link rel="stylesheet" type="text/css" href="css/main.css">
  <link rel="stylesheet" href="css/animate.css">
  <link rel="preconnect" href="https://fonts.gstatic.com">
  <link href="https://fonts.googleapis.com/css2?family=Oswald&display=swap" rel="stylesheet">


  <!-- Optional JavaScript -->
  <!-- jQuery first, then Popper.js, then Bootstrap JS -->
  <script src="js/jquery-3.5.1.min.js" type="text/javascript"></script>
  <script src="js/main.js" type="text/javascript"></script>
  <script src="bootstrap.min.js" type="text/javascript"></script>
  <script src="fontawesome.min.js" type="text/javascript"></script>
  <script src="fontawesome.js" type="text/javascript"></script>

  <link rel="preconnect" href="https://fonts.gstatic.com">
  <link href="https://fonts.googleapis.com/css2?family=Roboto+Slab&display=swap" rel="stylesheet">

  <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js" integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" crossorigin="anonymous"></script>
  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js" integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous"></script>

  <script src="js/wow.js"></script>
  <script>
    new WOW().init();
  </script>
  <script src="js/wow.min.js"></script>
  <script>
    new WOW().init();
  </script>

</head>

<body>
  <section id="cover">
    <div class="container">
      <div class="row">
        <nav class="navbar navbar-expand-sm fixed-top">
          <a class="navbar-brand" href="#"><img src="images/logo.png" width="50" height="50">ootstrap</a>
          <button class="navbar-toggler navbar-toggler-right" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
            <span class="navbar-toggler-icon"></span>
          </button>
          <div class="collapse navbar-collapse justify-content-end" id="navbarNav">
            <ul class="navbar-nav ">
              <li class="nav-item">
                <a class="nav-link" href="#cover">HOME<span class="sr-only">(current)</span></a>
              </li>
              <li class="nav-item">
                <a class="nav-link" href="#services">SERVICES</a>
              </li>
              <li class="nav-item">
                <a class="nav-link" href="#portfolio">PORTFOLIO</a>
              </li>
              <li class="nav-item">
                <a class="nav-link" href="#aboutus">ABOUT</a>
              </li>
              <li class="nav-item">
                <a class="nav-link" href="#contact">CONTACT</a>
              </li>
            </ul>
          </div>
        </nav>
      </div>
      <div class="row text-center">
        <div class="col-sm-12">
          <h1>Welcome to Our Studio!</h1>
        </div>
        <div class="col-sm-12">
          <h2>IT'S NICE TO MEET YOU</h2>
        </div>
        <div class="col-sm-12">
          <a href="#portfolio"><button type="button" class="btn btn-primary btn-lg">TELL ME MORE</button></a>
        </div>
      </div>
    </div>
  </section>

  <section id="services">
    <div class="container">
      <div class="services-top text-center wow fadeInDown" data-wow-duration="2s">
        <h3>SERVICES</h3>
        <p>Proin iaculis purus consequat sem cure.</p>
      </div>

      <div class="services-content text-center">
        <div class="row">
          <div class="col-md-4 wow fadeInLeft data-wow-duration=2s">
            <div class="item-circle">
              <i class="fas fa-leaf fa-3x"></i>
            </div>
            <h4>E-Commerce</h4>
            <p>Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.</p>

          </div>
          <div class="col-md-4 wow fadeInUp data-wow-duration=2s ">
            <div class="item-circle">
              <i class="fas fa-desktop fa-3x"></i>
            </div>
            <h4>Responsive Web</h4>
            <p>Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.</p>

          </div>
          <div class="col-md-4 wow data-wow-duration=2s fadeInRight ">
            <div class="item-circle">
              <i class="fas fa-lock fa-3x"></i>
            </div>
            <h4>Web Security</h4>
            <p>Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.</p>

          </div>

        </div>
      </div>

    </div>
  </section>

  <section id="portfolio" >
    <div class="container  text-center">
      <div class="portfolio-top">
        <h3>OUR PORTFOLIO</h3>
        <p>Proin iaculis purus consequat sem cure</p>
      </div>
      <div class="portfolio-contant">
        <div class="row">
          <div class="col-md-4">
            <div class="card">
              <img src="images/card1.jpg" alt="portfolio1" height="291px" width="310">
              <div class="card-block">
                <h4 class="card-title">Ebony &amp; Ivory</h4>
                <p class="card-subtitle text-muted">Branding</p>
              </div>
            </div>
          </div>
          <div class="col-md-4">
            <div class="card">
              <img src="images/card2.jpg" alt="portfolio2" height="291px" width="310">
              <div class="card-block">
                <h4 class="card-title">Smart Stationary</h4>
                <p class="card-subtitle text-muted">Print Design</p>
              </div>
            </div>
          </div>
          <div class="col-md-4">
            <div class="card">
              <img src="images/card3.jpg" alt="portfolio3" height="291px" width="310">
              <div class="card-block">
                <h4 class="card-title">Clever Poster</h4>
                <p class="card-subtitle text-muted">Print Design</p>
              </div>
            </div>
          </div>
        </div>
        <div class="row" >
          <div class="col-md-4">
            <div class="card">
              <img src="images/card4.jpg" alt="portfolio4" height="291px" width="310">
              <div class="card-block">
                <h4 class="card-title">Vinyl Record</h4>
                <p class="card-subtitle text-muted">Branding</p>
              </div>
            </div>
          </div>
          <div class="col-md-4">
            <div class="card">
              <img src="images/card5.jpg" alt="portfolio5" height="291px" width="310">
              <div class="card-block">
                <h4 class="card-title">Treehouse Template</h4>
                <p class="card-subtitle text-muted">Branding</p>
              </div>
            </div>
          </div>
          <div class="col-md-4">
            <div class="card">
              <img src="images/card6.jpg" alt="portfolio6" height="291px" width="310">
              <div class="card-block">
                <h4 class="card-title">Burned Log</h4>
                <p class="card-subtitle text-muted">Branding</p>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </section>

  <section id="aboutus">
    <div class="container">
      <div class="aboutus-top text-center">
       <h3>ABOUT US</h3>
       <p>Proin iaculis purus consequat sem cure</p>
     </div>
     <div class="aboutus-contant">
      <div class="row">
        <div class="col-md-4 text-right">
          <h4>JULY 2010</h4>
          <h4>Our Humble Beginnings</h4>
          <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
            tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
            quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
            consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
            cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
          proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
        </div>
        <div class="col-md-4 text-center">
          <img src="images/circle.jpg" width="170px" height="170px">
        </div>
      </div>
      <div class="row">
        <div class="col-md-4 offset-sm-4 text-center">
          <img src="images/circle.jpg" width="170px" height="170px">
        </div>
        <div class="col-md-4">
          <h4>JANUARY 2011</h4>
          <h4>Our Humble Beginnings</h4>
          <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
            tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
            quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
            consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
            cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
          proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
        </div>
      </div>
      <div class="row">
        <div class="col-md-4 text-right">
          <h4>JULY 2010</h4>
          <h4>Our Humble Beginnings</h4>
          <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
            tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
            quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
            consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
            cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
          proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
        </div>
        <div class="col-md-4 text-center">
          <img src="images/circle.jpg" width="170px" height="170px">
        </div>
      </div>
      <div class="row">
        <div class="col-md-4 offset-sm-4 text-center">
          <img src="images/circle.jpg" width="170px" height="170px">
        </div>
        <div class="col-md-4">
          <h4>JULY 2010</h4>
          <h4>Our Humble Beginnings</h4>
          <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
            tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
            quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
            consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
            cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
          proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
        </div>
      </div>

      <div class="row">
        <div class="col-md-4 offset-md-4">
          <div class="about-circle">
            <p class="yellow-text">
              OUR<br>
              STORY<br>
              CONTINIOUS<br>
            ...</p>
          </div>
        </div>
      </div>
    </div>
  </div>
</section>

<section id="team">
  <div class="container">

    <div class="team-top text-center">
      <h3>OUR AMAZING TEAM</h3>
      <p>Proin iaculis purus consequat sem cure</p>
    </div>
    <div class="team-contant">
      <div class="row">
        <div class="col-md-4 text-center">
          <div class="team-circle"></div>
          <h4>Kimberly Thompson</h4>
          <h5>Marketer</h5>
          <ul class="social-icons">
            <li class="social-icon">
              <i class="fab fa-twitter"></i>
            </li>
            <li class="social-icon">
              <i class="fab fa-facebook-f"></i>
            </li>
            <li class="social-icon">
              <i class="fab fa-pinterest-p"></i>
            </li>
          </ul>

        </div>
        <div class="col-md-4 text-center">
          <div class="team-circle"></div>
          <h4>Rico Massimo</h4>
          <h5>Coder</h5>
          <ul class="social-icons">
            <li class="social-icon">
              <i class="fab fa-twitter"></i>
            </li>
            <li class="social-icon">
              <i class="fab fa-facebook-f"></i>
            </li>
            <li class="social-icon">
              <i class="fab fa-pinterest-p"></i>
            </li>
          </ul>

        </div>
        <div class="col-md-4 text-center">
          <div class="team-circle"></div>
          <h4>Uku Mason</h4>
          <h5>Graphic Designer</h5>
          <ul class="social-icons">
            <li class="social-icon">
              <i class="fab fa-twitter"></i>
            </li>

            <li class="social-icon">
              <i class="fab fa-pinterest-p"></i>
            </li>
          </ul>
          

        </div>
      </div>
      <div class="row text-center">
        <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit,<br>sed do eiusmod
        tempor incididunt ut labore et dolore magna aliqua.</p>
      </div>  
    </div>
  </div>
  

</section>


<section id="partner">
  <div class="container">
    <div class="row text-center">
      <div class="col-md-3">
        <img src="images/envato.png" width="250px" height="100px">
      </div>
      <div class="col-md-3">
        <img src="images/envato.png" width="250px" height="100px">
      </div>
      <div class="col-md-3">
        <img src="images/envato.png" width="250px" height="100px">
      </div>
      <div class="col-md-3">
        <img src="images/envato.png" width="250px" height="100px">
      </div>
    </div>
  </div>
</section>

<section id="contact">
  <div class="container">
    <div class="contact-top text-center">
      <h3>CONTACT US</h3>
      <p>Proin iaculis purus consequat sem cure</p>
    </div>
    <div class="contact-contant">
      <form class="goldenform" role="form" method="post">
        <div class="row">
          <div class="col-md-6">
            <div class="form-element">
              <input type="text" class="goldenformelement" id="name" name="name" placeholder="YOUR NAME*">
            </div>
            <div class="form-element">
              <input type="email" class="goldenformelement" id="email" name="email" placeholder="YOUR EMAIL*">
            </div>
            <div class="form-element">
              <input type="text" class="goldenformelement" id="subject" name="subject" placeholder="SUBJECT">
            </div>
          </div>
          <div class="col-md-6">
            <div class="form-element">
              <textarea class="golden-form-element" name="message" placeholder="YOUR MESSAGE*"></textarea>
            </div>
          </div>
        </div>
        <div class="row justify-content-center">
          <div class="form-element">
            <input type="submit" class="btn btn-primary btn-lg" id="submit" name="submit" value="SEND MESSAGE">
          </div>
        </div>
      </form>
    </div>
  </div>
</section>

<section id="footer">
  <div class="container">
    <div class="row">
      <div class="col-md-4">
        <p>&copy;Copyright 2021 FreebiesXpress.com</p>
      </div>
      <div class="col-md-4 text-center">
        <ul class="company-social-icons">
          <li class="company-social-icon">
            <i class="fab fa-twitter"></i>
          </li>
          <li class="company-social-icon">
            <i class="fab fa-facebook-f"></i>
          </li>
          <li class="company-social-icon">
            <i class="fab fa-pinterest-p"></i>
          </li>
          <li class="company-social-icon">
            <i class="fab fa-google-plus"></i>
          </li>
        </ul>

      </div>
    </div>
  </div>      
</div>
</section>

   <!--
<h1>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</h1>


<h2>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</h2>
<br>
<h3>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</h3>

<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>


<h2>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</h2>
<br>
<h3>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</h3>

<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>


<h2>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</h2>
<br>
<h3>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</h3>

<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>


<h2>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</h2>
<br>
<h3>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</h3>
-->
</body>
</html>